### PR TITLE
More Improvements

### PR DESCRIPTION
**Pull request by giovanniorigins**

### CODE REVIEW

File: resources/assets/js/components/donate.vue [hunk 1]
```diff
@@ -452,13 +452,12 @@
...
             },
             submit(){
-                this.$refs.donationSpinner.show();
                 let data = {
                     amount: this.amount,
                     anonymous: this.anonymous,
                     currency: 'USD', // determined from card token
                     description: 'Donation to ' + this.title,
-                    comment: null,
+                    // comment: null,
```
Comment (nkeena) [inline resources/assets/js/components/donate.vue]: Why is this data property commented out?

File: resources/views/admin/financials/donors/show.blade.php [hunk 2]
```diff
@@ -109,7 +109,7 @@
                        id="{{ $donor->id }}"
                        user_id="{{ auth()->user()->id }}"
                        :per_page="3"
-                       :can-modify="{{ auth()->user()->can('modify-notes') }}">
+                       :can-modify="{{ auth()->user()->can('modify-notes')?1:0 }}">
```
Comment (nkeena) [inline resources/views/admin/financials/donors/show.blade.php]: There must be a better way to check permissions. I saw a suggestion to use a custom directive @can in a forum post. That could be really useful. 

Example
`<button @can('edit_notes')>Edit</button>`

File: resources/assets/js/components/donate.vue (hunk 1 above)
Comment (giovanniorigins) [reply resources/assets/js/components/donate.vue]: i think i commented that out to test donation

Comment (giovanniorigins) [reply resources/assets/js/components/donate.vue]: not to mention it isnt used anywhere else

Comment (nkeena) [reply resources/assets/js/components/donate.vue]: Oh wow you're right! No comment field.

Comment (giovanniorigins) [reply resources/assets/js/components/donate.vue]: Was there supposed to be one?

**Pull request merged**